ote/src/clj/ote/services/transport.clj

```diff
@@ -541,9 +540,90 @@
   [db operator-id]
   (http/transit-response (fetch-operator-users db {:operator-id operator-id})))
 
+(defn add-user-to-operator [email db new-member requester operator]
+  (let [group (create-member! db (:user_id new-member) (::t-operator/ckan-group-id operator))
+        ;; Ensure that we do not send email to test user
+        updated-member (if (= "user.userson@example.com" (:user_email new-member))
```

Comment:
Hard-coded values should be put to named symbols. Tests and code could then refer to same symbol, allowing updating value to one row.
Those constant symbols should be defined e.g. in cljc/ote.config/something.

Would logic readability be improved by moving the test user detection logic (one row, I admit) to separate function to make it more obvious what's happening?

Reply:
Added email-config.cljc file for these hard coded addresses